README.md

```diff
@@ -157,4 +157,18 @@
 	1. Incontri
 	2. Supplementari e rigori
 	3. Somma punta
-	4. Ex criterio UEFA gol trasferta
\ No newline at end of file
+	4. Ex criterio UEFA gol trasferta
+
+### Cambio regolamento
+
+#### Proprietari principali
+* Si definiscono **principali proprietari** coloro che, segnalati ad inizio campionato, amministrano principalmente la squadra
+* I principali proprietari vengono identificati su Github nel gruppo "proprietari" e sono i responsabili della revisione di ogni cambiamento proposto mediante Pull Request
+* Non puó esserci piú di un principale proprietario per squadra
+
+#### Come proporre un cambio di regolamento
+
+* Una proposta di cambio di regolamento é rappresentata da una richiesta di Pull Request su questa repository
+* Le proposte di cambio possono pervenire durante qualsiasi momento del campionato
+* le proposte di cambio vengono revisionate solo durante un'asta di fantamercato (iniziale o riparazione)
```

Comment:
secondo me solo all'asta estiva si fa la review, perchè non può cambiare il regolamento in corsa

Reply:
@jackthebeast praticamente stai dicendo che qualsiasi cosa verrà proposta in corso d'opera o dimenticanza non verrà valutata. giusto?

Reply:
a maggior ragione allora non ha molto senso scrivere che si aspetta una delle aste per decidere